demo/app/components/demo-map/demo-map.component.ts

```diff
@@ -25,6 +25,7 @@ export class DemoMapComponent {
 			fullscreenButton : false,
 			baseLayerPicker : false,
 			animation : false,
+			shouldAnimate: false,
```

Comment:
according to cesium docs the default value of viewer.shouldAnimate is false: https://cesiumjs.org/Cesium/Build/Documentation/Viewer.html